### DISCUSSION TITLE
Hacker Calculator App

### DISCUSSION THREAD

**Discussion by alexmarincu** [flet-dev/flet#2165] [Show and tell]
Hi flet.dev community. I recently discovered [flet](https://flet.dev/) and I think it's awesome. Thanks to the contributors.
I made a small app using the framework that evaluates expressions and provides hexadecimal and binary representations for integer results: [Hacker Calculator](https://github.com/alexmarincu/hacker-calculator). I have prepared Windows and Linux executables using flet pack ([releases](https://github.com/alexmarincu/hacker-calculator/releases)). Please check it out if you have the time.

Screenshot:
![image](https://github.com/flet-dev/flet/assets/28675077/241fbf33-ba32-45b5-adb5-11eef7054e13)

> NOTE
> For the assets, I had to do a little trick related to pyinstaller, to properly load them in the executable versions (based on this StackOverflow question: [link](https://stackoverflow.com/questions/31836104/pyinstaller-and-onefile-how-to-include-an-image-in-the-exe-file)). See snippets below. Please let me know if there is a better alternative.

![image](https://github.com/flet-dev/flet/assets/28675077/6268432b-67a2-4889-a933-95e4d5d0cb07)
![image](https://github.com/flet-dev/flet/assets/28675077/bfd225d7-e7da-429a-8084-b838efa873d7)

**Comment by ndonkoHenri** [comment]
Nice app.
For your issue, [`flet pack`](https://flet.dev/docs/cli/pack) has an optional parameter `--add-data` which you could use to specify your assets directory.
By the way, did you set `assets_dir` as such: `ft.app(target=main, assets_dir="assets") ` ?

**Reply by alexmarincu** [reply]
Hi and thanks for your answer. I am using `--add-data` to add the assets to the bundle. Before trying the workaround I mentioned, I also tried setting the `assets_dir`. The assets were working fine when running the app through the python interpreter, but in the executable made with pyinstaller, they weren't loading. Anyway, the `_MEIPASS` solution works for the time being.
